src/ess/amor/load.py

```diff
@@ -141,11 +141,19 @@ def load_amor_ch_frequency(ch: RawChopper[RunType]) -> ChopperFrequency[RunType]
 
 def load_amor_sample_rotation(fp: Filename[RunType]) -> SampleRotation[RunType]:
     (mu,) = load_nx(fp, "NXentry/NXinstrument/master_parameters/mu")
+    # We're only reading the first value here.
+    # That seems to do the right thing in the test cases that we've seen so far.
+    # It is probably not the right way to do it in all cases,
```

Comment:
Do we know if it is right, or is it just what Jochen's code does? I suggest to simply document this as something like "this is what software X by Y does".

Reply:
If it is right or not depends on if the value is truly time dependent or not. For the files that I have received to make this comparison I think its reasonable to assume it is right because that is how it is done in Jochens code.

But to know for sure why two values are written in the log instead of one, I'd have to ask Johchen.